### ISSUE TITLE
Possible incorrect url being used for verificationbucket

### ISSUE THREAD

**Issue by fabiancook** [aws-amplify/amplify-cli#2012]
**Describe the bug**
The url for the s3 bucket seems incorrect, when I look at the AWS console I see

```
http://${resourcePrefix}verificationbucket-${process.env.ENV}.s3-website.${region}.amazonaws.com
```

Rather than the coded

```
http://${resourcePrefix}verificationbucket-${process.env.ENV}.s3-website-${region}.amazonaws.com
```

The difference is a dot or dash before the region. 

You can see the line of code here https://github.com/aws-amplify/amplify-cli/blob/3cdbad863ad4febde47e56209d6026cddb344044/packages/amplify-category-auth/provider-utils/awscloudformation/triggers/CustomMessage/verification-link.js#L17 

**To Reproduce**
Use `amplify auth add` with a verification link

**Expected behavior**

A dot is used before the region. 

```
http://${resourcePrefix}verificationbucket-${process.env.ENV}.s3-website.${region}.amazonaws.com
```

**Comment by attilah** [comment]
@fabiancook Thanks for reporting the issue, what you found is a bug, but not that simple as it seems. As of the website endpoint documentation, the separator is region dependent.

Documentation: https://docs.aws.amazon.com/AmazonS3/latest/dev/WebsiteEndpoints.html

We'll need to use the table [here](https://docs.aws.amazon.com/general/latest/gr/rande.html#s3_website_region_endpoints) to implement the correct functionality.

Would you be interested in submitting a PR for this?

**Comment by attilah** [comment]
Fixed with [#2052](https://github.com/aws-amplify/amplify-cli/pull/2052)
